bento-sample-app/src/main/AndroidManifest.xml

```diff
@@ -9,7 +9,8 @@
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
         android:theme="@style/AppTheme">
-        <activity android:name=".MainActivity">
+        <activity android:name=".MainActivity"
+            android:exported="true">
```

Comment:
don't need  `android:exported="true"` here?

> If "true", the activity is accessible to any app, and is launchable by its exact class name.
If "false", the activity can be launched only by components of the same application, applications with the same user ID, or privileged system components. This is the default value when there are no intent filters.


https://developer.android.com/guide/topics/manifest/activity-element#exported

Reply:
emm ok never mind, "A launchable activity must be exported as of Android 12, which also makes it available to other apps."